### DISCUSSION TITLE
Error Processing (scaling and converting) the image

### DISCUSSION THREAD

**Discussion by RjAsap** [upscayl/upscayl#698] [Q&A]
![report bug](https://github.com/upscayl/upscayl/assets/148029165/f0a66525-7d41-462e-945b-5d8ec13766b6)

Why everytime i try to upscale an image, this will be happen?

**Comment by NayamAmarshe** [comment]
Try renaming the image to something else

**Reply by RjAsap** [reply]
i,m sorry, but it's still same

**Reply by aaronliu0130** [reply]
Follow the pinned #392
